ndcube/ndcube_sequence.py

```diff
@@ -8,7 +8,7 @@
 from ndcube import utils
 from ndcube.visualization.descriptor import PlotterDescriptor
 
-__all__ = ['NDCubeSequence']
+__all__ = []
```

Comment:
Same here as in other alls, now these are only normally discoverable from the main ndcube import.